app/controllers/ops_controller/settings/common.rb

```diff
@@ -182,7 +182,11 @@ def pglogical_save_subscriptions
       task_opts = {:action => "Save subscriptions for global region", :userid => session[:userid]}
       queue_opts = {:class_name => "MiqPglogical", :method_name => "save_global_region",
                     :args       => [subscriptions_to_save, subsciptions_to_remove]}
-      ActiveRecord::Base.yaml_column_permitted_classes |= [subscriptions_to_save.first.class, subsciptions_to_remove.first.class]
+      if ActiveRecord.respond_to?(:yaml_column_permitted_classes)
+        ActiveRecord.yaml_column_permitted_classes       = YamlPermittedClasses.app_yaml_permitted_classes | [subscriptions_to_save.first.class, subsciptions_to_remove.first.class]
+      else
+        ActiveRecord::Base.yaml_column_permitted_classes = YamlPermittedClasses.app_yaml_permitted_classes | [subscriptions_to_save.first.class, subsciptions_to_remove.first.class]
+      end
```

Comment:
Note, this change is similar to other changes in other repositories and I'll be removing the conditional across the board when we move to rails 7 and drop 6.1.